_pages/Dashboard/components/Cards/index.js

```diff
@@ -1,5 +1,7 @@
 import styles from './styles/cards.module.css';
 
+const lang = { mw: 'Modern Warfare', cw: 'Cold War', psn: 'Playstation', wz: 'War Zone', mp: 'Multiplayer' };
+
 function Card({ title, activityType, platform, username, imageUrl }) {
```

Comment:
This can be defined using an ES6 arrow function, so instead of:

`function Card({ title, activityType, platform, username, imageUrl }) {`

You could use:

`const Card = ({ title, activityType, platform, username, imageUrl }) => {`

You have already done this in `Statsboard/index.js` and would be nice for it to be consistent across the board